monapi/urls.py

```diff
@@ -23,5 +23,5 @@
 
 urlpatterns = [
     path('', include(router.urls)),
-    path('admin/', admin.site.urls),
```

Comment:
adminnya jangan diapus dulu kali ya wkwkk, keknya masih perlu buat nanti

Reply:
updated, thank you